app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -133,27 +134,33 @@ export class MergeConflictsDialog extends React.Component<
 
   private renderConflictedFile(
     path: string,
-    conflicts: number,
+    conflictStatus: ConflictStatus,
     editorName: string | undefined,
     onOpenEditorClick: () => void
   ): JSX.Element {
-    const humanReadableConflicts = this.calculateConflicts(conflicts)
-    const message =
-      humanReadableConflicts === 1
-        ? `1 conflict`
-        : `${humanReadableConflicts} conflicts`
-    return (
-      <li className="unmerged-file-status-conflicts">
-        <Octicon symbol={OcticonSymbol.fileCode} className="file-octicon" />
-        <div className="column-left">
-          <PathText path={path} availableWidth={200} />
-          <div className="file-conflicts-status">{message}</div>
-        </div>
-        <Button onClick={onOpenEditorClick}>
-          {this.editorButtonString(editorName)}
-        </Button>
-      </li>
-    )
+    if (conflictStatus.kind === 'binary') {
+      return <div>TODO</div>
```

Comment:
can we remove this? i don't see a reason to surface this in the UI (except for dev debugging).

Reply:
Want it to return `null` or render a successful file?

Reply:
I made this return `null` for a binary diff in 651781d. LMK if you have any further input.